deployments/helm/hephaestus/templates/_helpers.tpl

```diff
@@ -36,35 +45,60 @@ Return the service account name.
 Returns a unified list of image pull secrets.
 */}}
 {{- define "hephaestus.imagePullSecrets" -}}
-{{- include "common.images.pullSecrets" (dict "images" (list .Values.controller.image .Values.logProcessor.image) "global" .Values.global) }}
+{{- include "common.images.renderPullSecrets" (dict "images" (list .Values.controller.image .Values.logProcessor.image .Values.buildkit.image) "global" .Values.global) }}
 {{- end }}
 
 {{/*
-Returns the logfile directory.
+Return whether or not securityContext.seccompProfile field is supported.
 */}}
-{{- define "hephaestus.logfileDir" -}}
-/var/log/hephaestus
+{{- define "hephaestus.supportsSeccompGA" -}}
+{{- semverCompare ">1.19-0" .Capabilities.KubeVersion.Version }}
```

Comment:
with 5.2, we only support v1.19 and higher so its your call

Reply:
that's good to know. I feel fine removing this then.